doc/manual/src/glossary.md

```diff
@@ -7,6 +7,59 @@
     translated into low-level *store derivations* (implicitly by
     `nix-env` and `nix-build`, or explicitly by `nix-instantiate`).
 
+    [derivation]: #gloss-derivation
+
+  - [store derivation]{#gloss-store-derivation}\
+    A [derivation] represented as a `.drv` file in the [store].
+    It has a [store path], like any [store object].
+
+    See [`nix show-derivation`](./command-ref/new-cli/nix3-show-derivation.md) (experimental) for displaying the contents of store derivations.
```

Comment:
I don't see any `new-cli` directory in https://github.com/fricklerhandwerk/nix/tree/doc-instantiate/doc/manual/src/command-ref

When I click on the link from a Github page, I land - as expected - on a 404

Is this intended? If so, would you mind explain to me why this will work _in fine_ on the documentation/website?

Reply:
These files are generated in the [Makefile](https://github.com/NixOS/nix/blob/f91dc023f27ea17362274fcdf30e362423b080f8/doc/manual/local.mk#L51-L53) through a [Nix expression](https://github.com/NixOS/nix/blob/f91dc023f27ea17362274fcdf30e362423b080f8/doc/manual/generate-manpage.nix) that is fed by running the `nix` executable.

Reply:
For my own culture: is there a CI check that all links are valid in the generated documentation?